examples/src/main/scala/org/apache/spark/examples/snappydata/structuredstreaming/CDCExample.scala

```diff
@@ -25,35 +25,35 @@ import scala.language.postfixOps
 import scala.reflect.io.Path
 
 /**
- * An example showing CDC usage with SnappyData structured streaming
+ * An example showing CDC usage with SnappyData streaming sink.<br>
  *
- * <p></p>
- * To run the example in local mode go to your SnappyData product distribution
- * directory and type following command on the command prompt
- * <pre>
- * bin/run-example snappydata.StructuredStreamingCDCExample
- * </pre>
- * <p></p>
  * To run this on your local machine, you need to first run a Netcat server <br>
  * `$ nc -lk 9999`
  * <p>
  * Sample input data:
- * {{{
+ * <pre>
  * 1,user1,23,0
  * 2,user2,45,0
  * 1,user1,23,2
  * 2,user2,46,1
- * }}}
+ * </pre>
+ *
+ * To run the example in local mode go to your SnappyData product distribution
+ * directory and execute the following command:
+ * <p>
+ * `bin/run-example snappydata.structuredstreaming.CDCExample`
+ * <br><br>
  *
  */
-object StructuredStreamingCDCExample{
+// scalastyle:off println
+object CDCExample{
 
   def main(args: Array[String]) {
     // reducing the log level to minimize the messages on console
     Logger.getLogger("org").setLevel(Level.ERROR)
     Logger.getLogger("akka").setLevel(Level.ERROR)
 
-    println("Initializing a SnappySesion")
+    println("Initializing a SnappySession")
     val checkpointDirectory = "/tmp/StructuredStreamingCDCExample"
```

Comment:
@vatsalmevada I don't think it is a good idea to use /tmp for checkpoints in a production app. Make sure you make a note of this in your docs or comments.

Reply:
@jramnara for new examples I have accepted the checkpoint directory an argument. This is an existing example and will update this too. Thanks for pointing out.